server/character.lua

```diff
@@ -6,6 +6,17 @@ end
 
 ---@param source Source
 local function giveStarterItems(source)
+    local getInv = function(source) return exports.ox_inventory:GetInventory(source) ~= false end
+    local i = 0
+    local invCreated = getInv(source)
+    while not invCreated and i < 100 do
+        i += 1
+        Wait(100)
+        invCreated = getInv(source)
+    end
```

Comment:
repeat until? Or do the event based method

Reply:
When I say event based method I'm referring to https://github.com/Qbox-project/qbx_core/pull/196#discussion_r1351502762